gz_ros2_control/include/gz_ros2_control/gz_ros2_control_plugin.hpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
```

Comment:
For some reason, this file is being treated as a new file instead of a rename of the old `ign` version. Would it be possible to fix that? Maybe starting with a `git mv`? I think it would make reviews and future forward ports easier.